BUILD.bazel

```diff
@@ -1879,7 +1879,6 @@ xnnpack_cc_library(
         "@FP16",
         "@FXdiv",
         "@clog",
-        "@cpuinfo",
```

Comment:
This change is already landed in bf182e886b8e0df20f3c34ed8172baa4d88ad446

Reply:
Synced to head.